src/components/KeyboardShortcutsModal.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import {View} from 'react-native';
+import _ from 'underscore';
+import HeaderWithCloseButton from './HeaderWithCloseButton';
+import Text from './Text';
+import Modal from './Modal';
+import CONST from '../CONST';
+import styles from '../styles/styles';
+import withWindowDimensions, {windowDimensionsPropTypes} from './withWindowDimensions';
+import withLocalize, {withLocalizePropTypes} from './withLocalize';
+import compose from '../libs/compose';
+import KeyboardShortcut from '../libs/KeyboardShortcut';
+
+
+const propTypes = {
+    /** prop to fetch screen width */
+    ...windowDimensionsPropTypes,
+
+    /** props to fetch translation functions */
+    ...withLocalizePropTypes,
+};
+
+class KeyboardShortcutsModal extends React.PureComponent {
```

Comment:
I don't think we want to use `PureComponent` here because it won't update when the window dimensions or locale change.

```suggestion
class KeyboardShortcutsModal extends React.Component {
```